### PR TITLE
SRCH-3543 Updated footer link

### PR DESCRIPTION
**Pull request by yogesh27**
## Summary
- Updated a footer link from "Federalist" to "Cloud.gov Pages" on the system login page footer
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@
     t.text "metadata"
     t.string "service_name", null: false
     t.bigint "byte_size", null: false
-    t.string "checksum", null: false
+    t.string "checksum"
```
Comment (danswick) [inline db/schema.rb:32]: Is this change supposed be included?

**Review by danswick — CHANGES_REQUESTED** [review]
The updated link looks good to me, @yogesh27, but I wasn't sure if the `schema.rb` change (noted inline) was intentional.

File: db/schema.rb (hunk 1 above)
Comment (yogesh27) [reply db/schema.rb:32]: Hi @danswick - db/schema.rb got updated as a part of running the DB migrations when I took a pull from the main branch.

**Review by danswick — APPROVED** [review]
👍 looks good to me. Thanks, @yogesh27!

**Pull request merged**